contents/docs/llm-analytics/generations.mdx

```diff
@@ -1,24 +1,102 @@
 ---
 title: Generations
-availability:
-  free: full
-  selfServe: full
-  enterprise: full
 ---
 
 import GenerationEvent from "./_snippets/generation-event.mdx"
+import NotableGenerationProperties from "./_snippets/notable-generation-properties.mdx"
 
-Generations are an event that capture an LLM request. The [generations tab](https://app.posthog.com/llm-analytics/generations) lists them along with the properties autocaptured by PostHog like the person, model, total cost, token usage, and more.
+Generations are events that capture LLM calls and their responses. They represent interactions and conversations with an AI model. Generations are tracked as `$ai_generation` events and can be used to create and visualize [insights](/docs/product-analytics/insights) just like other PostHog events.
 
-When you expand a generation, it includes the properties, metadata, a conversation history, the role (system, user, assistant), input content, and output content.
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/ai_generations_f687da8aaa.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/ai_generations_dark_51d8bae273.png"
+  alt="$ai_generation events"
+  classes="rounded"
+/>
+<Caption>
+  View recent AI generation events in the <b>Activity</b> tab
+</Caption>
+
+The **LLM analytics** > [**Generations** tab](https://app.posthog.com/llm-analytics/generations) displays a list of generations, along with a preview of key autocaptured properties. You can filter and search for generations by various properties.
 
 <ProductScreenshot
-  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/llma_generations_8a94f2cd01.png"
-  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/llma_generations_dark_50e206f940.png"
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/llm_generations_b12119af33.png"
```

Comment:
This columns showing the input and output is behind a feature flag that is only used internally at the moment. I'm not sure that we want to show that yet. WDYT?